### ISSUE TITLE
Migration not working

### ISSUE THREAD

**Issue by Plevuus** [koillection/koillection#674]
First of all, thank you for making this application. It looks great, works nice and I expect to be using it with pleasure. 

This is a bit of a mixed bug-report and support request. I'm unable to transfer my data form one instance to another.

Background: 
I'm currently running Koillection on docker on my windows computer. There is one user (myself), one collection, and  ~210 items. Each item has ~20 attributes and one image. I now want to transfer that to my 'homelab' computer running Ubuntu LTS 22.04 Server and Docker.

Pre-work:
Before trying to transfer I did some preparation and some sanity checks:

1. Updated Koillection on my PC to the latest version and check if it still worked. It did.
2. Created a docker-compose file that also exposes a port for the mySQL server to the outside world.
3. Up-ed the docker compose on the homelab server and checked if it worked. It did work.
4. Checked if I could connect to the database with HeidiSQL in my windows computer to the docker mySQL server on the homelab server. I could connect and browse the site.
5. Stopped and removed the docker containers on the homelab server and deleted the files on the volumes

Attempt one:
In my first attempt I followed the approach as suggested in issue [#504](https://github.com/koillection/koillection/issues/504):

1. Created the containers on my homelab server using the docker-compose file.
2. Stopped all the containers on my windows computer
3. Copied al files from the volumes on my windows computer to the volumes on the homelab server. (except mysql.sock)
4. Started the mySQL container on the homelab server. It was running.
5. Tried to connect with HeidiSQL to the mySQL server. That failed. No connection possible. (10061)
6. Started the Koillection container on the homelab server and visited the webpage. The webpage did not open and the containers stopped.

I also tried with starting and stopping the containers on the homelab server  in-between step 2 and 3. The result was the same. I tries several times and also 'incrementally' moved the files from the SQL volume on the windows PC to the homelab server. It turned out that the mysql.idb was causing the issue.

Attempt two:
In my second attempt I followed a different approach:

1. Created the containers on my homelab server using the docker-compose file.
2. Exported the SQL from the koillection instance running on the windows computer using the web interface.
3. Stopped all the containers on my windows computer.
4. Copied al files from the Koilection volumes on my windows computer to the volumes on the homelab server. I did NOT copy the files in the mySQL volumes.
5. Started the containers on the home-lab server and stopped the koillection container.
6. Used heidiSQL and connected to the mySQL server on the homelab computer.
7. Deleted the Koillection database.
8. Re-created the Koillection database using: CREATE DATABASE `koillection` /*!40100 COLLATE 'utf8mb4_0900_ai_ci' */
9. Executed the downloaded SQL (as UTF-8).
10. Enabled the user in the database.
11. Started the Koillection container on the homelab server and opened the webpage.
12. That was working and I could also log in as the existing user. I saw my collection and in the collection in grid view I saw images of the items. In list view I also saw attributes. However when I clicked on any of the items I get a 'critical error'

I have no clue on how to successfully do the migration. Any help is appreciated!

**Comment by Plevuus** [comment]
Update (on attempt method 2):
I noticed that in the database on the server there was no data in the koi_choice_list table. With HeidiSQL I copied the data  from that table from the database on the windows computer to the database  on the homelab server and after that the issue is gone. I can open the individual items!
It seems that the choice_list is not in the SQL export. It seems to be missing from src/Service/DatabaseDumper.php (but I must admit this is way beyond my knowledge level)
